### PR TITLE
bump relay to v7

### PR DESCRIPTION
**Pull request by wenzowski**
closes a whole bunch of dependabot PRs if stable

### CODE REVIEW

File: app/pages/form-builder.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React, {Component} from 'react';
 import {Container, Row, Col} from 'react-bootstrap';
-import Header from '../components/Header';
+// Import Header from '../components/Header';
```
Comment (matthieu-foucault) [inline app/pages/form-builder.js]: Remove this page? It's not used, and broken

File: .circleci/config.yml [hunk 2]
```diff
@@ -194,6 +194,19 @@ jobs:
...
 
+  compile:
+    machine:
+      image: ubuntu-1604:201903-01
+    working_directory: ~/cas-ciip-portal/app
+    steps:
+      - attach_workspace:
+          at: ~/
+      - run:
+          name: Audit Dependencies
```
Comment (matthieu-foucault) [inline .circleci/config.yml]:
```suggestion
          name: Make a production build
```

File: .circleci/config.yml [hunk 3]
```diff
@@ -194,6 +194,19 @@ jobs:
...
+          command: |
+            source ~/.asdf/asdf.sh
+            yarn build # ensure next.js can compile a production build
```
Comment (matthieu-foucault) [inline .circleci/config.yml]:
```suggestion
            yarn build:relay
            yarn build:next # ensure next.js can compile a production build
```
This is what we're doing on pathfinder. See the s2i assemble script

**Comment by matthieu-foucault** [comment]
You'll need to update the snapshots, looks like relay 7 adds more forwardrefs

File: .circleci/config.yml (hunk 2 above)
Comment (wenzowski) [reply .circleci/config.yml]: yessir

File: .circleci/config.yml (hunk 3 above)
Comment (wenzowski) [reply .circleci/config.yml]: good point, the schema is checked in a separate, isolated step

File: app/pages/form-builder.js (hunk 1 above)
Comment (wenzowski) [reply app/pages/form-builder.js]: done

**Comment by wenzowski** [comment]
rebased on develop

**Pull request merged**